### DISCUSSION TITLE
How to run in MacOS?

### DISCUSSION THREAD

**Discussion by cmbail** [qarmin/czkawka#984] [General]
I successfully compiled on M1 Mac using the instructions at https://github.com/qarmin/czkawka/blob/master/instructions/Instruction.md

After compilation, the program ran successfully.

However, after quitting, I can't figure out how to run it again.

There are no instructions on how to run at https://qarmin.github.io/czkawka/instructions/Instruction.html

I have tried "chmod +x ./czkawka_gu" then "./czkawka_gui" in Terminal, but get "zsh: permission denied"

Ideally, I would make a shortcut in the Applications folder so I can run it from MacOS GUI.

**Comment by qarmin** [comment]
If you compiled app from source, then binary should work fine(steps are almost identical between mac and linux)
```
cd czkawka # Cloned repo folder
cd target
cd release # Depending on compilation options, this may be debug/release or even different folder name
chmod +x ./czkawka_gui # This is only needed once
./czkawka_gui 
```
If still this not works(that would be strange), then compilation again should work fine(cargo should see that there is nothing to recompile and will run instantly in second and later runs)
```
cd czkawka
cargo run --release --bin czkawka_gui
```

**Reply by cmbail** [reply]
The first set worked. Thank you!

My mistake was that I was working on zkawka_gui in the parent and not the release folder.

**Comment by avibathula** [comment]
When the pre-built binaries are made available on homebrew, why not make the fully packaged app available on Homebrew?
